Gemfile

```diff
@@ -39,7 +39,8 @@ gem 'jemalloc'
 gem 'customerio'
 
 group :development, :test do
-  gem 'travis-migrations', git: 'https://github.com/travis-ci/travis-migrations'
+  gem 'travis-migrations', git: 'https://github.com/travis-ci/travis-migrations', branch: 'igor-logs-migrations'
+  gem 'sequel'
```

Comment:
travis-migrations _does_ require sequel, but for some reason bundler did not pick it up. :-/